src/get_timezone.py

```diff
@@ -13,21 +13,30 @@
 api = APIService()
 
 
-def get_timezone_from_str(tz_string: Optional[str]) -> Optional[datetime.timezone]:
+def get_timezone_utc_format(txt_pattern: Optional[str]) -> Optional[datetime.timezone]:
+    """Returns datetime.timezone based on string in format UTC+00:00 or NONE."""
+    if txt_pattern is None:
+        return None
+    tz_pattern = r"^(UTC)?(?P<sign>[-+]?)(?P<hours>(0?[1-9])|(1[0-4]))(?P<minutes>:0{1,2}|:15|:30|:45)?$"
+    tz_result = re.search(tz_pattern, txt_pattern, flags=re.IGNORECASE)
+    if tz_result is None:
+        return None
+
+    minutes = tz_result.group("minutes")
+    minutes = 0 if minutes is None else minutes
```

Comment:
```suggestion
    minutes = 0 if minutes is None else int(minutes)
```
раз у нас тут может быть число, то пусть оно 100% будет